File: asciinema/commands/help.py
class HelpCommand(object):

    def execute(self):
        print HELP_TEXT


HELP_TEXT = '''usage: asciinema [-h] [-y] [-c <command>] [-t <title>] [action]

Asciicast recorder+uploader.

Actions:
 rec              record asciicast (this is the default when no action given)
 auth             authenticate and/or claim recorded asciicasts

Optional arguments:
 -c command       run specified command instead of shell ($SHELL)
 -t title         specify title of recorded asciicast
 -y               don't prompt for confirmation
 -h, --help       show this help message and exit
 -v, --version    show version information'''

File: commands/help.go
package commands

import "fmt"

type HelpCommand struct{}

func (c *HelpCommand) Execute(args []string) error {
	fmt.Println(`usage: asciinema <command> [options]

Asciinema terminal recorder.

Commands:
   rec       Record asciicast
   auth      Assign local API token to asciinema.org account
   version   Show version information (also -v and --version)

Run "asciinema <command> -h" to see the options available for the given command.`)

	return nil
}
